examples/taskruns/old-home-volume.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: tekton.dev/v1alpha1
+kind: TaskRun
+metadata:
+  generateName: old-home-volume-
+spec:
+  taskSpec:
+    steps:
+    - name: write
+      image: ubuntu
+      command: ['bash']
+      args: ['-c', 'echo some stuff > /builder/home/stuff']
+    - name: read
+      image: ubuntu
+      command: ['cat']
+      args: ['/builder/home/stuff']
+    - name: override-homevol
+      image: ubuntu
+      command: ['bash']
+      # /builder/home/stuff *doesn't* exist, because our specified volumeMount
+      # overrides it.
+      args: ['-c', '[[ ! -f /builder/home/stuff ]]']
+      volumeMounts:
+      - name: empty
+        mountPath: /builder/home
```

Comment:
question for my own self reasons (aka #1438 ) : how useful do you find it to be able to declare volumeMounts for every step and is this something you'd want if you could define a volumeMount for the entire Task instead?

Reply:
I think that's possible today with the step template

Reply:
Yeah, I think that's what `stepTemplate` and `task.spec.volumes` is for.

Reply:
I mean it's _possible_ but do you feel like you _want_ to do it - i.e. would you be happy with just having a volume that is mounted for the entire Task and `volumeMounts` on a step didnt work?

if im not making any sense then just ignore me and ill ask you both again once I have an example :D